src/legacy/ui/public/courier/fetch/handle_response.tsx

```diff
@@ -17,14 +17,14 @@
  * under the License.
  */
 
-
 import React from 'react';
-import { toastNotifications } from '../../notify/toasts';
 import { i18n } from '@kbn/i18n';
 import { EuiSpacer } from '@elastic/eui';
+import { toastNotifications } from '../../notify/toasts';
 import { ShardFailureOpenModalButton } from './components/shard_failure_open_modal_button';
+import { Request, ResponseWithShardFailure } from './components/shard_failure_types';
 
-export function handleResponse(request, response) {
+export function handleResponse(request: any, response: any) {
```

Comment:
`any`

Reply:
The types for these (`request`/`response`) should really come from Elasticsearch. I could switch them to use the `SearchParams` and `SearchResponse` from `elasticsearch` but since they're incomplete it will mess up some of the usages. (This goes for `highlight` and `aggs` too.) In the interim I can create a `SearchRequest` and `SearchResponse` type that are just `any` and point to them and in a follow-up PR we can try to fill out these types a little better. Thoughts?